pkgs/applications/networking/instant-messengers/discord/darwin.nix

```diff
@@ -16,7 +16,9 @@ stdenv.mkDerivation {
     runHook postInstall
   '';
 
-  postInstall = lib.strings.optionalString (openasar != null) ''
+  postInstall = lib.strings.optionalString withOpenASAR ''
     cp -f ${openasar} $out/Applications/${desktopName}.app/Contents/Resources/app.asar
   '';
+
+  buildInputs = lib.optionals withOpenASAR [ unzip ];
```

Comment:
Unless this is necessary to build Discord itself whehn `withOpenASAR = true` (that I find highly unlikely, since `withOpenASAR` seems to only copy something to the final output), them this is unnecessary.

The reference to `unzip` in OpenASAR is sufficient.